mobile/src/main/res/layout/search_frag.xml

```diff
@@ -249,6 +249,25 @@
                     android:text="@string/search_color_identity_title" />
             </LinearLayout>
 
+            <LinearLayout
+                style="@style/forty_eight"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+                <TextView
+                    style="@style/text.medium"
+                    android:layout_width="wrap_content"
+                    android:layout_height="match_parent"
+                    android:text="Is Commander"
```

Comment:
Please use string resources for the UI, not hardcoded strings. This is untranslatable.

Reply:
It is not yet pushed, but I have this fixed.

Reply:
This has been pushed as of now.